be/src/runtime/fragment_mgr.cpp

```diff
@@ -763,13 +763,7 @@ Status FragmentMgr::exec_plan_fragment(const TExecPlanFragmentParams& params, Fi
             _pipeline_map.insert(std::make_pair(fragment_instance_id, context));
             _cv.notify_all();
         }
-        auto st = context->submit();
```

Comment:
Assume task1, task2, task3 exist in this context, if we submit task1 successfully and task2 failed, will submitted task1 be processed properly?

Reply:
If task2 submit fail, `In PipelineFragmentContext::submit` will cancel the context and set `_total_tasks` to 1.
If `_closed_tasks == _total_tasks` represents task1 is close, we try to exec `_close_action`.
if `_closed_tasks != _total_tasks`, task1 will exec `_close_action`.